base/page_base.py

```diff
@@ -0,0 +1,59 @@
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+class BaseClass(object):
+    """
+    The class we need to call all pages
+
+    """
+
+    def __init__(self, driver):
+        self.driver = driver
+        self.wait = WebDriverWait(self.driver, 30)
+
+    def wait_for_element(self, selector):
+        """
+        item to be selected
+
+        """
+        return self.wait.until(EC.element_to_be_clickable(selector))
+
+    def presence_for_element(self, selector):
+        """
+        Showing the item
```

Comment:
açıklama yanlış. -> presence of given selector diye düzeltip, selector'e params ekleyelim